tool/tbot/main.go

```diff
@@ -69,7 +69,7 @@ func Run(args []string, stdout io.Writer) error {
 
 	startCmd := app.Command("start", "Starts the renewal bot, writing certificates to the data dir at a set interval.")
 	startCmd.Flag("auth-server", "Address of the Teleport Auth Server or Proxy Server.").Short('a').Envar(authServerEnvVar).StringVar(&cf.AuthServer)
-	startCmd.Flag("token", "A bot join token, if attempting to onboard a new bot; used on first connect.").Envar(tokenEnvVar).StringVar(&cf.Token)
+	startCmd.Flag("token", "A bot join token or full path to file with token value, if attempting to onboard a new bot; used on first connect.").Envar(tokenEnvVar).StringVar(&cf.Token)
```

Comment:
"Full path" is not as clear as _absolute path_.

But I wonder, does it actually have to be an absolute path?

Reply:
We tend to use "Full Path" or just path.  You can do relative paths AFAIK, just that if you're running in a service you better run it in the right place so full path recommended I'd say.  Only error messages on the term absolute I could fine.

`--pid-file            Full path to the PID file. By default no PID file will be created`
or just path
`  -c, --config              Path to a configuration file [/etc/teleport.yaml]`

Reply:
Please see comments @zmb3 when you get a chance